packages/react-components/react-utilities/src/hooks/useId.ts

```diff
@@ -37,7 +37,9 @@ export function useId(prefix: string = 'fui-', providedId?: string): string {
   const _useId: () => string | undefined = (React as never)['use' + 'Id'];
```

Comment:
Are you sure `useId` can return `undefined`? The react docs don't mention this, you could avoid having the null chaining 

https://beta.reactjs.org/reference/react/useId

Reply:
yes, that's the case. I'll fix it.